app/views/spree/admin/orders/_form.html.haml

```diff
@@ -2,6 +2,9 @@
   - if @line_item.try(:errors).present?
     = render :partial => 'spree/shared/error_messages', :locals => { :target => @line_item }
 
+  - if @order.insufficient_stock_lines.any?
```

Comment:
TLDR version:
```suggestion
  - if !@order.completed? && @order.insufficient_stock_lines.any?
```
I think that'll be enough :point_up: 

We might want a little bit more in terms of test coverage though.